README.md

```diff
@@ -175,10 +175,10 @@ canvas.toDataURL.mockReturnValueOnce(
 
 ## Contributors
 
-* [@hustcc](https://github.com/hustcc)
-* [@jtenner](https://github.com/jtenner)
-* [@evanoc0](https://github.com/evanoc0)
+- [@hustcc](https://github.com/hustcc)
+- [@jtenner](https://github.com/jtenner)
```

Comment:
move yourself to first by the comment and contribute.

Reply:
You are the primary contributor. This was *your* idea and *your* repo in the first place. I merely filled in the details and assertions.

I don't need to be placed primary on the list.